notebooks/en/enterprise_cookbook_dev_spaces.md

```diff
@@ -0,0 +1,88 @@
+
+<img src="https://cdn-lfs-us-1.huggingface.co/repos/13/3d/133d8ca2460bf82ba2bdbe928d91a6c780364a6d0cf9005087db081cca492c02/4b119b0ae20e55127488b1f0317413b36491936696b354bd8d1012ea28a1e06f?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27enterprise-jupyterlab-creation-1.png%3B+filename%3D%22enterprise-jupyterlab-creation-1.png%22%3B&response-content-type=image%2Fpng&Expires=1717861243&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcxNzg2MTI0M319LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy11cy0xLmh1Z2dpbmdmYWNlLmNvL3JlcG9zLzEzLzNkLzEzM2Q4Y2EyNDYwYmY4MmJhMmJkYmU5MjhkOTFhNmM3ODAzNjRhNmQwY2Y5MDA1MDg3ZGIwODFjY2E0OTJjMDIvNGIxMTliMGFlMjBlNTUxMjc0ODhiMWYwMzE3NDEzYjM2NDkxOTM2Njk2YjM1NGJkOGQxMDEyZWEyOGExZTA2Zj9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoifV19&Signature=G3L195ww4hShsdRVo6hXSQ9kXvfJ0S73l4ifLsRoSCmPHVp7GUbDbtFiWakrJVuDVjXYEWGb7zbGDxoVLdIhEE04OmNLPFo6bxjAiktN-Gb5qPUJUq4NkZAtvrI2kMep0bSwbx4Ua8nvXQDfRXJ8K7J0FXXczRAYw2VQS9avEqTdD7l1Vgp7vIM6NXhuU4fClKkQf2XNdIxAFLEsrwT8O0KrEBE46Lemj6J6gixqLz5VL6la5QZf3gJEwhd7CZSkYAFUGs8w1BtliuVvqIc0YktD1tK338WIlYTQ6KgBgnRYcZaZa3pjmv0M-BzFXGth-60Y68TsQQAML024kuL-4w__&Key-Pair-Id=KCD77M1F0VK2B" width="450">  
+
+- **Choosing your hardware**: You can choose from a wide range of hardware from free CPUs to A100 GPUs. When setting up the Space, we recommend you choose the free basic CPU. You can switch to better paid hardware once you need it (see available hardware and prices [here](https://huggingface.co/pricing)). 
+- **Persistent storage**: It is important to attach persistent storage to the Space, so that all the files you create (code, models, data) is also saved when the Space is paused or reset. You can always increase the disk space in the settings later when necessary. All persistet data is stored in the `/data` directory ([docs](https://huggingface.co/docs/hub/en/spaces-storage)).
+- **Set your password**: Once the Space is created, it will require a password for logging into JupyterLab. This password is defined with the `JUPYTER_TOKEN` Space secret. If you do not define a password here, the default password is "huggingface". 
+- **Dev Mode**: Dev Mode is a feature for Enterprise Hub subscribers which enables you to SSH into any HF Space. Activate this to be able to connect your local VS Code for remote development on the Space's cloud hardware (this can also be switched on/off later). See the preview docs [here](https://huggingface.co/dev-mode-explorers). 
```

Comment:
Should we mention this is a beta feature or is that just for non-PRO and non-Enterprise users?

```suggestion
- **Dev Mode**: Dev Mode is a feature for Enterprise Hub subscribers that enables you to SSH into any HF Space. Activate this to connect your local VS Code for remote development on the Space's cloud hardware (this can also be switched on/off later). See the preview docs [here](https://huggingface.co/dev-mode-explorers). 
```